readme/INSTALL.md

```diff
@@ -1,5 +1,23 @@
-# Installation
-
+# Installation (CPU mode)
+
+install anaconda
+open anaconda powershell
+conda install m2-base
+install cuda toolkit 12.4, download cudnn 9.2 and unzip contents into C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.4
+To find cuda <> cudnn compatibility use: https://docs.nvidia.com/deeplearning/cudnn/latest/reference/support-matrix.html
+conda create --name rock-tech python
+conda activate rock-tech
+git clone git@github.com:nubinary/Rocky-CenterNet.git
+cd Rocky-CenterNet
+pip install -r requirements.txt
+cd ./src/lib/models/networks/DCNv2
+python setup.py build develop
+cd ..\..\..\external\
```

Comment:
Did you mean to use forward slashes here?

Reply:
Concerning the feedback on the specific steps, I've spaced them out, and added coco api as a separate step.
For the rest, I think I'll wait for Sina's feedback, and then make modifications.